### ISSUE TITLE
Port documentation from Wiki: ShortText Field

### ISSUE THREAD

**Issue by herflis** [SenseNet/sensenet.github.io#332]
Port old article from wiki and refresh them for sensenet 7. The new article should be an .md file in the /_docs folder. Old screenshots should be re-created on the new admin-ui if it is possible and placed into the /docs/images folder.
[admin.sensenet.com](https://admin.sensenet.com)
name: BuiltIn\admin
password: admin
repository url: https://dev.demo.sensenet.com

The .md file should have the following header:

```
---
title: %title%
source_url: %url of the file on github%
category: Development
version: v7.0.0
tags: %possible tags as an array%
---
```

(you can use [the docs here](https://github.com/SenseNet/sensenet.github.io/tree/master/_docs) as example)

[ShortText field](http://wiki.sensenet.com/ShortText_Field)

Please change the old wiki urls as if they were available on github (e.g. https://wiki.sensenet.com/content_templates sould be changed to /content-templates)

**Comment by cyberShaw** [comment]
I'd like to work on this. So I have to update the documentation for ShortText Field with the new header and add the new screenshots into admin/ui, am I correct?

**Comment by herflis** [comment]
Hi @cyberShaw ,
you have to create a new .md file here in the /_docs folder, copy the content of the ShortText field wiki article and put it in the .md file, format the text as it was formatted on the Wiki and add it a header as it is mentioned above 🙂 If you need more help, feel free to ask!

**Comment by cyberShaw** [comment]
Sure thing! I'll work on it and send a PR :)
Also, there is already a docs file for shorttext-field. Should I let it be or delete it?

**Comment by cyberShaw** [comment]
So I have done two commits on my forked repository.
One where I updated the screenshots for HowToCreateContent on the new adminUI : 15596255e0e287498ec15dac93701d1408152b0f
The other where I added docs for shorttext-field : 2aba5a159d4ae35551636b994724e7e0d5c49248

If that's a go, I'll send in a PR

**Comment by shreyasbapat** [comment]
Fixed in #339
